### PR TITLE
Final Version of the Smashing Magazine Design Teardown

### PR DESCRIPTION
**Pull request by SpaYco**
## Building with Responsive Design: The Next Web Clone
- Added NavBar
- Added Main Section
- Added Footer
- Responsive to 2 extra sizes
- No Linter Issues

[Live Demo](https://raw.githack.com/Eshy10/The-Next-Web-Clone/feature-page/index.html)

### CODE REVIEW

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Really fantastic job @Spayco and @Eshy10 
It is all coming together and shining! Please consider my comments below, make the required changes and request another review :handshake: 

#### Horizontal Scroll
You seem to have a little bug with one of the elements of the page, shooting out beyond the browser's maximum width. This may be as a result of a margin or padding you have set up. One of the easiest ways I have found to correct this issue is to set the default `box-sizing` property to `border-box` as follows `*{box-sizing: border-box}`. Please read some more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing).

#### Responsiveness
Your project could use a little more responsiveness, please look a little more closely at the sizes of elements, padding of sections, margins, and general element positionings. These are very key as they will affect the look, feel, and more importantly the usability of your sites. I went ahead to attach a few images of areas where I noticed issues with responsiveness, however, please do a full scan of your page just in case I missed or left something out.
screenshot
screenshot
screenshot

### Summary
You have come a long way and your hard work is glaring. Please consider my comments below, so your project can shine even more than it already does! Look out for any element that is causing the horizontal scroll, and look into your mobile responsiveness thoroughly, and you should be good to go. I wish you all the best in this and all other projects. See you at the top! :rocket:

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

File: fonts.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+@font-face { font-family: ARS Maquette; font-style: italic; font-weight: 400; src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/ars-maquette-regular-italic.eot); src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/ars-maquette-regular-italic.eot#iefix) format("embedded-opentype"),url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/ars-maquette-regular-italic.woff) format("woff"),url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/ars-maquette-regular-italic.ttf) format("truetype"); }
+
+@font-face { font-family: ARS Maquette; font-style: normal; font-weight: 700; src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/ars-maquette-bold-alt.eot); src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/ars-maquette-bold-alt.eot#iefix) format("embedded-opentype"),url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/ars-maquette-bold-alt.woff) format("woff"),url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/ars-maquette-bold-alt.ttf) format("truetype"); }
```
Comment (euqueme) [inline fonts.css]: delete these unused repeated fonts (lines #3 to #5)

File: fonts.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+@font-face { font-family: Avalon; font-style: normal; font-weight: 700; src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/avalon-bold.eot); src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/avalon-bold.eot#iefix) format("embedded-opentype"),url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/avalon-bold.woff) format("woff"),url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/avalon-bold.ttf) format("truetype"); }
+
+@font-face { font-family: TNW-Avalon; font-style: normal; font-weight: 700; src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/TNW-Avalon-Bold.otf); src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/tnw-avalon-bold.ttf); }
```
Comment (euqueme) [inline fonts.css]: Delete this unused font (line #9)

File: fonts.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+@font-face { font-family: Trump; font-style: normal; font-weight: 700; src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/Trump-Gothic-East-Bold.ttf); }
+
+@font-face { font-family: Graphik; font-style: normal; font-weight: 400; src: url(https://cdn4.tnwcdn.com/assets/fonts/graphik/graphik-regular.woff) format("woff"),url(https://cdn4.tnwcdn.com/assets/fonts/graphik/graphik-regular.woff2) format("woff2"); }
+
+@font-face { font-family: Graphik-Medium; font-style: normal; font-weight: 500; src: url(https://cdn4.tnwcdn.com/assets/fonts/graphik/graphik-medium.woff) format("woff"),url(https://cdn4.tnwcdn.com/assets/fonts/graphik/graphik-medium.woff2) format("woff2"); }
```
Comment (euqueme) [inline fonts.css]: Delete these unused fonts (line #13 to line #17)

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

## Hello @Eshy10 and @Spayco :balloon: :lollipop: 

### Amazing job so far :medal_sports: your project is looking more similar to the original :art:  every time and has good use of semantic HTML and the Readme file is stellar :star: Still, there are a few changes to work on before we can give you the approval:

## Small Screens

**navbar**

- [x] All links in the navbar are hidden and the TWN logo has to appear in the leftmost part of it

- [x] it's missing a hamburger icon in the rightmost part of the navbar

- Yours

image

- Original

image

**Latest Fundings**

- [x] the last article in this section should be hidden (the original only has 3, you have 4)

**Footer**

- [x] The text in the footer should be centered and place in 3 rows just like the original

- Yours

image

- Original

image

## Medium Screens (width > 768px)

**navbar**

- [x] most of the links in the navbar are showing and the TWN logo has to appear in the leftmost part of it

- Yours

image

- Original

image

**Main Articles**

- [x] it has one article in the leftmost part of the page and 2 in the rightmost one

- Yours

image


- Original

image

**Latest news**

- [x] this section has 3 articles per row in this screen size

- Yours

image

- Original

image

**Latest Fundings**

- [x] Only one row with 3 articles in it

- Yours

image

- Original

image

**Categories (Apps, GERA, Tech, etc) section**

- [x] must have 2 articles per row

- Yours

image

- Original

image

**Latest Deals**

- [x] has 3 articles per row

- Yours

image

- Original

image

## Large Screens (width > 1023px)

**All sections**

- [x] articles should have more space in between them so they take at least 95% of the page width

- Yours

image

- Original

image

## CSS good practices

- [x] use or delete the unused fonts I have spotted in your code bellow  :arrow_down_small: 

### That's it team! :raised_hands: 

### Please :pray:  make the changes :memo:  and submit back for a code review when you are done :rocket: 

any questions? ask me in slack **@ Maru**

Happy Coding! :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Nice job team! @Spayco and @Eshy10 
You went on to fix the issues I pointed out during my last review, however, the issues @euqueme rightly pointed out on the tablet screen size(<=768px) still persist. In addition to those, there are some new issues with overflowing elements on the page.

#### Horizontal Scroll
You seem to have a little bug with one of the elements of the page, shooting out beyond the browser's maximum width. This may be as a result of a margin or padding you have set up. One of the easiest ways I have found to correct this issue is to set the default `box-sizing` property to `border-box` as follows `*{box-sizing: border-box}`. Please read some more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing). Please see attached screenshots
screenshot
screenshot
screenshot

### Summary
You did a great job tackling the issues I pointed out in my last review. However, please take some time to consider my comments above in conjunction with what @euqueme rightly pointed out. This project is all about responsiveness and as such a lot of attention is paid to it in the reviews. Take some time and go through each section, examine it against the original page, make appropriate changes and request another review. I wish you the very best! See you at the top :medal_military: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,529 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./reset.css">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./fonts.css">
+    <script src="https://kit.fontawesome.com/f02bbd63cf.js" crossorigin="anonymous"></script>
```
Comment (MauricioRobayo) [inline index.html:10]: This file is not loading:

> GET https://kit.fontawesome.com/f02bbd63cf.js net::ERR_CONNECTION_CLOSED

Please make sure it is working fine before merging 💪.

File: fonts.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+@font-face { font-family: ARS Maquette; font-style: normal; font-weight: 400; src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/ars-maquette-regular.eot); src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/ars-maquette-regular.eot#iefix) format("embedded-opentype"),url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/ars-maquette-regular.woff) format("woff"),url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/ars-maquette-regular.ttf) format("truetype"); }
+
+@font-face { font-family: Shentox; font-style: normal; font-weight: 500; src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/shentox-medium.eot); src: url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/shentox-medium.eot#iefix) format("embedded-opentype"),url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/shentox-medium.woff2) format("woff2"),url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/shentox-medium.woff) format("woff"),url(https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/shentox-medium.ttf) format("truetype"); }
```
Comment (MauricioRobayo) [inline fonts.css]: These resources are being blocked by the CORS policy on the browser:

> Access to font at 'https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/fonts/shentox-medium.woff2' from origin 'https://raw.githack.com' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'https://thenextweb.com' that is not equal to the supplied origin.

**Review by MauricioRobayo — APPROVED** [review]
Congratulations, the project looks awesome and all the requirements are met!

Go ahead and merge 🚀.

Status: __Approve++__.

---

Please take note that some of your resources are not loading due to CORS policy. The fonts that you are requesting are being blocked by the browser, If you want to use them, you might want to host them yourselves... but make sure you have the rights to do so in that case. Also, check the font awesome that you're loading, it seems to be failing:

image

Please check these issues on the `Network` tab of the `Developer Tools` of the browser of your choice.

File: fonts.css (hunk 5 above)
Comment (Spayco) [reply fonts.css]: weird, for some reason, I don't get that error

**Pull request merged**